### PR TITLE
adding MaskedInput component for an example to code sandbox

### PR DESCRIPTION
**Pull request by mrebb**
I have added a  code sandbox example for `MaskedInput` component that covers both time and phone number format examples.

### CODE REVIEW

File: src/MaskedInput.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            },
+            { fixed: ' ' },
+            {
+              length: 2,
+              options: ['am', 'pm'],
+              regexp: /^[ap]m$|^[AP]M$|^[aApP]$/,
+              placeholder: 'ap',
+            },
+          ]}
+          value={text}
```
Comment (ericsoderberghp) [inline src/MaskedInput.js]: Can we call this `time`, to be clearer what it is?

Comment (mrebb) [reply src/MaskedInput.js]: Yes, Shall I modify and push the code again? Thanks Eric for quick review.

Comment (ericsoderberghp) [reply src/MaskedInput.js]: that would be great

**Comment by ShimiSun** [comment]
fixes https://github.com/grommet/grommet/issues/2647

**Pull request merged**